CHANGELOG.md

```diff
@@ -10,7 +10,7 @@
 
 ### Fixed
 * [ALFREDAPI-463](https://xenitsupport.jira.com/browse/ALFREDAPI-463): Fix quotation marks in searches being improperly escaped. Search queries can now be escaped properly. E.g. \"Compas Format\" instead of \\\"Compas Format\\\" as was needed previously.
-
+* [ALFREDAPI-466](https://xenitsupport.jira.com/browse/ALFREDAPI-466): Add input normalization to `PersonService#GetPerson()`
```

Comment:
For a commit, this is a good message, but a changelog should be user-oriented -> `Fix broken /v2/people/-me- call` for example